src/index.js

```diff
@@ -294,61 +295,65 @@ class PopoverTooltip extends React.PureComponent<Props, State> {
           onRequestClose={this.props.onRequestClose}
           transparent
         >
-          <Animated.View style={[
-            styles.overlay,
-            this.props.overlayStyle,
-            { opacity: this.state.opacity },
-          ]}>
-            <TouchableOpacity
-              activeOpacity={1}
-              focusedOpacity={1}
-              style={styles.button}
-              onPress={this.toggle}
-            >
-              <Animated.View
-                style={[
-                  styles.tooltipContainer,
-                  this.props.tooltipContainerStyle,
-                  tooltipContainerStyle,
-                ]}
-              >
-                <View
-                  onLayout={this.onInnerContainerLayout}
-                  style={styles.innerContainer}
+          <View style={{flex: 1}} accessible={true} accessibilityLabel={this.props.accessibilityLabel}>
```

Comment:
@erickrmattos: Sorry for the delay on the review, this `{ flex: 1 }` (and the other on the TouchableOpacity) could be placed in the `Stylesheet.create` in the end of the file and used as a `styles.wrapper`, what do you think?

Reply:
Since it is only a prop in this case, what do you think about we start doing that in the future changes?

Reply:
Yeah, sure! 👍